nuclear/module/python/nuclear.py

```diff
@@ -284,8 +284,8 @@ def Reactor(reactor):
             for include in reaction[1].include_paths():
                 includes.add('#include "{}"'.format(include))
 
-        macro_guard = "{}_HPP".format(reactor_name.upper())
-        header_file = "{}.hpp".format(reactor_name)
+        macro_guard = "{}_H".format(reactor_name.upper())
```

Comment:
Didn't we change to `.hpp` files everywhere?

Reply:
Yeah, you missed this one. Hence why it's a minor fixes PR :P.
Although the NUbots version of this file overrides it so this won't ever run anyway

Reply:
This diff is changing the `hpp` to `h` though. I think I missed it on the other repo.

Reply:
Would be better to just fix it while we are here.

Reply:
Its correct here.

Reply:
Oh, looks like I messed this one up